### ISSUE TITLE
Support SELinux volume modes.

### ISSUE THREAD

**Issue by jfindley** [ansible/ansible-modules-core#2176]
##### ISSUE TYPE

Feature Idea
##### COMPONENT NAME

docker module
##### ANSIBLE VERSION

N/A
##### SUMMARY

Since docker 1.7, two additional volume modes are supported on top of 'ro' and 'rw': 'z' and 'Z'.  These set the SELinux mode of the volume to be accessible from every container or just this container, respectively.

See http://www.projectatomic.io/blog/2015/06/using-volumes-with-docker-can-cause-problems-with-selinux/ for more info on this.

The docker-py does seem to support this option - setting mode: Z works fine in my testing, but ansible is coded to only accept 'ro' or 'rw'.

**Comment by robynbergeron** [comment]
Hi @jfindley -- thanks for filing this feature request. 

@cove @softzilla @smashwilson @joshuaconner -- pinging you to let you know about this feature request for the docker module.

**Comment by robynbergeron** [comment]
Also -- pinging my lovely pals from the land of Atomic,  @jasonbrooks @dustymabe @nzwulfin -- in case any of you are interested in maybe making this happen or know someone who might be :)

**Comment by robynbergeron** [comment]
[module: cloud/docker/docker.py]

**Comment by dustymabe** [comment]
@robynbergeron Thanks for thinking of me. I don't have time right now to look at this but I'm interested in having this work in the future so I may be back soon!

**Comment by robynbergeron** [comment]
Hi @jfindley -- note that #2312 looks like it may be a fix for this -- it would be great to get your feedback there. 

@dustymabe -- thanks for the comment, just figured I'd give y'all a heads-up! :)

**Comment by alvaroaleman** [comment]
I think this can be closed, as a patch to add this feature is merged into devel already.

**Comment by ansibot** [comment]
@cove, @softzilla, @zfil, @ThomasSteinbach, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@cove, @softzilla, @zfil, @ThomasSteinbach, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@jfindley We appreciate you taking the time to file this issue, but the module you've opened the issue about has been deprecated.

We will consider critical bugfixes for deprecated modules, but all other types of requests will be closed.

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)
